client/my-sites/site-indicator/index.jsx

```diff
@@ -146,10 +135,9 @@ export class SiteIndicator extends Component {
 		);
 	}
 
-	recordEvent( event, total ) {
+	recordEvent() {
 		window.scrollTo( 0, 0 );
 		this.setState( { expand: false } );
```

Comment:
If we remove these parameters (`event`, `total`).
Should we remove them also from the calls?

https://github.com/Automattic/wp-calypso/blob/e4f9ed11816d51c78d2e4e42c138909e3928e4fd/client/my-sites/site-indicator/index.jsx#L144-L170

Reply:
@sejas Good catch. I did some refactoring with 7313ed11b5901c91a71676638ca3bfb70ec5bd12

Reply:
Great refactor!! ✨